suitcase/pizza_box/handlers.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, resource_path, chunk_size=1024):
             chunk = chunk[column_keys]
             self.chunks_of_data.append(chunk)
 
-    def __call__(self, chunk_num, column):
+    def __call__(self, chunk_num, column=0):
```

Comment:
Is this default value necessary? The only reason I can think of that a Handler would need a default value is if there are old Datum documents missing a parameter that was later added.

Reply:
Yes it is unfortunately. The reason is to prove backwards compatibility with the older version of the handler which doesn't know about this. Currently ISS doesn't have a `column` kwarg but QAS does.
The other option is to keep two different handlers. If we do go this route, we should do the same for eigerhabdler which is also following this pattern (eiger handler has two keys of which either is used depending on the version)

Reply:
(sorry for typos phone autocorrect)